app/controllers/api/application_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# rubocop:disable ClassAndModuleChildren
+class API::ApplicationController < ApplicationController
+  include Rails::Pagination
+
+  before_action :add_security_headers
+
+  def authenticate_user!
+    return become_active if logged_in? && adequate_scopes? && current_user.authorized_access_token?
+    render_forbidden
```

Comment:
If user isn't logged in, returns 403 with unauthorized message. Might modify in future to redirect to OAuth URL.

Reply:
:+1: